apps/web/src/components/PaginatedTable/helpers/blobs.tsx

```diff
@@ -0,0 +1,109 @@
+import { Link } from "~/components/Link";
+import { StorageIcon } from "~/components/StorageIcon";
+import type { DeserializedBlob } from "~/utils";
+import {
+  buildBlockRoute,
+  buildTransactionRoute,
+  formatBytes,
+  shortenAddress,
+  formatTimestamp,
+} from "~/utils";
+
+export const blobsTableHeaders = [
+  {
+    cells: [
+      {
+        item: "Versioned Hash",
+      },
+      {
+        item: "Transaction Hash",
+      },
+      {
+        item: "Block Number",
+      },
+      {
+        item: "Timestamp",
+      },
+      {
+        item: "Size",
+      },
+      {
+        item: "Storage",
+      },
+    ],
+  },
+];
+
+type BlobItem = Pick<
+  DeserializedBlob,
+  "versionedHash" | "commitment" | "size" | "dataStorageReferences" | "proof"
+> & {
+  blockHash: string;
+  blockNumber: number;
+  blockTimestamp: string;
+  txHash: string;
+};
+
+export const getBlobsTableRows = (blobs?: BlobItem[]) =>
+  blobs
+    ? blobs.map(
+        ({
+          versionedHash,
+          size,
+          dataStorageReferences,
+          txHash,
+          blockTimestamp,
+          blockNumber,
+        }) => ({
+          cells: [
+            {
+              item: shortenAddress(versionedHash, 8),
+            },
+            {
+              item: (
+                <div className="text-contentTertiary-light dark:text-contentTertiary-dark">
+                  <Link href={buildTransactionRoute(txHash)}>
+                    {shortenAddress(txHash, 8)}
+                  </Link>
+                </div>
+              ),
+            },
+            {
+              item: (
+                <div className="text-contentTertiary-light dark:text-contentTertiary-dark">
+                  <Link href={buildBlockRoute(blockNumber)}>{blockNumber}</Link>
+                </div>
+              ),
+            },
+            {
+              item: (
+                <div className="min-w-60 whitespace-break-spaces">
+                  {formatTimestamp(blockTimestamp)}
```

Comment:
Timestamp seems a little bit verbose. Can we show only the `x ago` part and omit the full date?

Reply:
Done! Added a `compact` param for the util